### DISCUSSION TITLE
Mobile version translation?

### DISCUSSION THREAD

**Discussion by aruto77** [FabulousCupcake/umamusume-db-translate#25] [Q&A] [answered]
is there a way to use this for mobile version?

**Comment by FabulousCupcake** [comment, marked as answer]
Sure, as long as you can replace `master.mdb`.  
This is easy to do in android (unsure if root is required) and probably impossible in ios unless jailbroken.

**Reply by aruto77** [reply]
where i can find this master.mdb, because i've searched in file explorer and i've not found this file anywhere
Is it root required to access this file?

**Reply by 7se7en** [reply]
~~I'm on the same boat as aruto77. Could I get some guidance on the steps, please?~~
Yes, root access is required.
But when loading the game, I get stuck at 60%. I'm probably failing some file integrity check. DMM doesn't check?

**Reply by aruto77** [reply]
in my case, after root access is granted i can access the file and after that i upload to the provided link and copy the downloaded file to the database directory and overwrite it. Don't forget to disable again the root access because if you didn't disable first, the game will not be played properly.
